src/riak_kv_vnode.erl

```diff
@@ -110,6 +110,21 @@
                 reqid :: term(),
                 target :: pid()}).
 
+-record(counter_state, {
```

Comment:
Naming is hard. I think the inline comments make up for any possible confusion about its purpose.